Base/usr/share/man/man2/posix_spawn.md

```diff
@@ -0,0 +1,47 @@
+## Name
+
+`posix_spawn` - launch a new process
```

Comment:
It's markdown, so functions that contain a `_` must have backticks, since markdown treats `_` even in the middle of a word as start of italics. Functions that don't contain underscores don't need this.

Reply:
No; <code>``</code> in Markdown emits the <code>&lt;code&gt;</code> HTML tags, and disables underscore (and star, and other special characters) handling as a side effect. The proper way to deal with underscores inside words is to escape them, like this: `posix\_spawn`

Reply:
Done.